src/extensions/codemirror/_codemirror.scss

```diff
@@ -13,6 +13,7 @@
         width: 100%;
         height: 290px;
         clip-path: unset !important; // we may check later why they set inset(0) now
+        background-color: $eccgui-color-primary-contrast;
```

Comment:
The correct value would be `$eccgui-color-textfield-background`, see https://github.com/eccenca/gui-elements/blob/next/src/components/TextField/textfield.scss#L13

Even better would be to set an own var like `$eccgui-color-codeeditor-background: $eccgui-color-textfield-background !default;` and then use this one in the rule.